floodviz/utils.py

```diff
@@ -0,0 +1,90 @@
+import requests
+
+"""
+    Takes a valid url and a dictionary of expected column header strings as keys (values do not matter).
+    parse_rdb then requests data from the service at url, and extracts data matching any header values.
+     
+    Data is returned as a list of dict objects, where each object represents a data point. 
+
+"""
+
+
+def parse_rdb(url, header_values):
```

Comment:
I was thinking of a much more generic parse function: Something like
parse_rdb(url, params) where url is the non query string part of the request and params are the query parameters without including the format for rdb (that would be added) by this function.  I would pass back all columns. Let the caller of the function decide what to do with the dictionary.